luigi/contrib/hive.py

```diff
@@ -297,7 +297,7 @@ def hiveconfs(self):
         * hive.exec.reducers.max (reducers_max)
         """
         jcs = {}
-        jcs['mapred.job.name'] = self.task_id
+        jcs['mapred.job.name'] = "'" + self.task_id + "'"
```

Comment:
`"'%s'" % self.task_id` would have been marginally more pythonic, but not blocking